### ISSUE TITLE
Allow full URL's, not only hostnames

### ISSUE THREAD

**Issue by derwasp** [fsprojects/SwaggerProvider#41]
### Description

It is not possible to specify the full URL (e.g.  `http://my.awesome.api/`) as a parameter.
The reason I need this is because I want to be able to specify both `http` and `https` endpoints whenever I want to.

Consider the following code:

```fsharp
[<Literal>]
let private awesomeJsonPath = __SOURCE_DIRECTORY__ + "/awesome.swagger.json"
type AwesomeApi = SwaggerProvider<awesomeJsonPath, "Content-Type=application/json">

let private client = new AwesomeApi("http://my.awesome.api/")

printfn "%A" client.AwesomeValueGet(42)
```

Currently this will crash.

### Expected behavior

`http`/`https` prefixes are respected and taken into account when the request is constructed

### Actual behavior

> An unhandled exception of type 'System.Net.WebException' occurred in FSharp.Core.dll
> Additional information: The remote name could not be resolved: 'http'

### Known workarounds

Use only hostname

**Comment by sergey-tihon** [comment]
You should be able to create two clients for two endpoints
```fsharp
[<Literal>]
let private awesomeJsonPath = __SOURCE_DIRECTORY__ + "/awesome.swagger.json"
type AwesomeApi = SwaggerProvider<awesomeJsonPath, "Content-Type=application/json">

let private clientHttp = new AwesomeApi("http://my.awesome.api/")
let private clientHttps = new AwesomeApi("https://my.awesome.api/")

printfn "%A" clientHttp.AwesomeValueGet(42)
printfn "%A" clientHttps.AwesomeValueGet(42)
```

**Comment by derwasp** [comment]
Please take a look a this screenshot:
![capture](https://cloud.githubusercontent.com/assets/10754345/17093223/421a0ddc-5248-11e6-8678-aab0faecfc80.PNG)

And that is what happens if Fiddler is launched:
![fiddler](https://cloud.githubusercontent.com/assets/10754345/17093345/fd3f6bde-5248-11e6-84a7-452d2cc6e06f.PNG)

SwaggerProvider used: 0.5.3

I guess [this](https://github.com/fsprojects/SwaggerProvider/blob/2184d9e625056c1ce2070f3beca36c6df91390ae/src/SwaggerProvider.DesignTime/OperationCompiler.fs#L57) is why.

**Comment by sergey-tihon** [comment]
Should be fixed in [v0.5.5](https://www.nuget.org/packages/SwaggerProvider/0.5.5)
